db/migrate/20201013170323_simplify_columns_in_notifications.rb

```diff
@@ -1,5 +1,3 @@
-# typed: true
-
```

Comment:
whoa - what is this ... and why change an older migration?

Reply:
This is a magic comment that Sorbet (type-checker) uses to determine how tightly to type-check a file. But since this was added, we have instructed Sorbet to IGNORE the entire db directory because the files are largely generated, which means when they are regenerated, we'd need to re-add the type comments manually. I removed this comment for consistency with the other files in db, so as not to suggest that type-checking is being done on this file.